history/history-network.md

```diff
@@ -338,5 +355,13 @@ Merkle proof with the `BlockHeader`'s block hash as leave and the
 `EpochRecord` digest as root. This digest is available in the
 `HistoricalHashesAccumulator`.
 
-As the `HistoricalHashesAccumulator` only accounts for blocks pre-merge, this proof can
-only be used to verify blocks pre-merge.
+As the `HistoricalHashesAccumulator` only accounts for blocks pre-merge, this proof MUST be used to verify blocks pre-merge (pre Paris fork).
+
+#### HistoricalRootsBlockProof
+
+The `HistoricalRootsBlockProof` is an SSZ container which holds two Merkle proofs as specified in the [SSZ Merke proofs specification](https://github.com/ethereum/consensus-specs/blob/dev/ssz/merkle-proofs.md#merkle-multiproofs).
+
+The container holds a chain of 2 proofs. This chain of proofs allows for verifying that an EL `BlockHeader` is part of the canonical chain. The only requirement is having access to the beacon chain `historical_roots`.
+The `historical_roots` is a [`BeaconState` field](https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#beaconstate) that is frozen since the Capella fork. The `HistoricalRootsBlockProof` MUST be used to verify blocks from TheMerge/Paris until the Capella fork.
```

Comment:
This is to minimal to be understandable. I'll need to add some further explanation here.